CONTRIBUTING.md

```diff
@@ -1,11 +1,40 @@
-# How to Contribute
+# Contributing to Kubeflow Spark Operator
 
-We'd love to accept your patches and contributions to this project. There are
+Welcome to the Kubeflow Spark Operator project. We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Code reviews
+## Follow the Code of Conduct
+
+Before any contribution you would like to make, please have a look at the [code of conduct](CODE_OF_CONDUCT.md), which details how contributors are expected to conduct themselves as part of the Kubeflow Spark Operator Community.
+
+## Code Contributions
+
+The Spark Operator project accepts code contributions via Github pull requests(PR), and this is the only way accepted to apply your changes to the Fluid project.
+
+You should submit a PR If any modifications is going to be applied to the Fluid project, including but not limited to:
+
+- Fix typos
+- Fix bugs
+- Fix or polish documents
+- Prune redundant codes
+- Add comments to codes for readability
+- Add missing test cases
+- Add new features or enhanced some feature
+- Refactor codes
```

Comment:
I also notice that you have this Developer guide on the website: https://www.kubeflow.org/docs/components/spark-operator/developer-guide/
Do you want to link it in this doc so users will understand how to get started with Spark Operator development?

Reply:
Done.